pootle/checks.py

```diff
@@ -267,11 +267,11 @@ def check_settings(app_configs=None, **kwargs):
         if markup_filter is not None:
             try:
                 if markup_filter == 'textile':
-                    import textile  # noqa
+                    import textile  # noqa pylint:disable=unused-variable
                 elif markup_filter == 'markdown':
-                    import markdown  # noqa
+                    import markdown  # noqa pylint:disable=unused-variable
                 elif markup_filter == 'restructuredtext':
-                    import docutils  # noqa
+                    import docutils  # noqa pylint:disable=unused-variable
```

Comment:
We already do.  I'm also anti, except where the places we need it are minor and the overall benefit is larger enough.  I.e. where we have no false positives other that these ones which we can tightly control and where we're operating slightly out of normal.

Of course suggestions to tighten the code are the other option.